### PR TITLE
Add same Opportunity onchange to Lead view

### PR DESCRIPTION
**Pull request by vrenaville**
Apply same on change modification done for Opportunity to Lead

### CODE REVIEW

**Comment by pedrobaeza** [comment]
I don't remember well, but I think that onchanges were removed on purpose. @dreispt, can it be?

File: crm_department/crm_view.xml [hunk 1]
```diff
@@ -31,6 +31,13 @@
       <field name="model">crm.lead</field>
       <field name="inherit_id" ref="crm.crm_case_form_view_leads" />
       <field name="arch" type="xml">
+        <xpath expr="/form/sheet/group/group[3]/div/field[@name='section_id']" position="attributes">
+          <attribute name="on_change">onchange_section_id(section_id)</attribute>
+        </xpath>
+	    <!-- Inactive on change, because the user is already changed by the section_id on change -->
+        <field name="user_id" on_change="on_change_user(user_id, context)" position="attributes">
```
Comment (dreispt) [inline crm_department/crm_view.xml:38]: You just need to use: `<field name="user_id" position="attributes">`

**Comment by dreispt** [comment]
@pedrobaeza I looked at the history, and I believe those have been @vrenaville 's doings.

**Comment by pedrobaeza** [comment]
OK then :+1:

**Comment by jgrandguillaume** [comment]
:+1:

**Pull request merged**